manifests/c/Codemao/TurtleEditor/2/2.6.1/Codemao.TurtleEditor.2.installer.yaml

```diff
@@ -0,0 +1,29 @@
+# Created with komac v2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: Codemao.TurtleEditor.2
+PackageVersion: 2.6.1
+InstallerLocale: zh-CN
+InstallerType: nullsoft
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+FileExtensions:
+- py
+- tur
+#AppsAndFeaturesEntries: --> By komac
+#- DisplayName: 海龟编辑器2 2.6.1
+#  Publisher: codemao
+Installers:
+- Architecture: x86
+  InstallerUrl: https://static.codemao.cn/turtle/client/release/%E6%B5%B7%E9%BE%9F%E7%BC%96%E8%BE%91%E5%99%A82-x32-2.6.1.exe
+  InstallerSha256: B76B16E52EC933C1938456148706804B8BFBED46BFD8D600A5B17C59749C310E
+  ReleaseDate: 2024-08-29
+- Architecture: x64
+  InstallerUrl: https://static.codemao.cn/turtle/client/release/%E6%B5%B7%E9%BE%9F%E7%BC%96%E8%BE%91%E5%99%A82-x64-2.6.1.exe
+  InstallerSha256: 7ACF1FC439C3161AFA7C76117B4B47A8B6A07A4693AB855FFDA51C5FBCE91177
+  ReleaseDate: 2024-08-28
```

Comment:
And here.
(About https://github.com/microsoft/winget-pkgs/labels/Manifest-Metadata-Consistency )